game/scripts/vscripts/libraries/timers.lua

```diff
@@ -257,7 +269,7 @@ function Timers:RemoveTimers(killAll)
 	end
 
 	Timers.timers = timers
-end
+end]]--
```

Comment:
It's not used as-is, but may be really helpful while debugging things in the console. If it not requires any changes, could you leave it? Though we don't use `v.persist`, so maybe it can be simplified to something like

```lua
function Timers:RemoveTimers()
	self.realTimeHeap = BinaryHeap("endTime")
	self.gameTimeHeap = BinaryHeap("endTime")
end
```

Or maybe even rename it to `Timers:InitializeHeap` and call in the `Timers:start()`, to be DRY